docs/installation/octopus-in-container/migrate-to-server-container-linux-from-windows-container.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Migrate to Octopus Server Linux Container from Windows Container
+description: A guide on how to migrate to the Octopus Server Linux Container from the Octopus Server Windows Container
+position: 3
+---
+
+The Octopus Server Windows Container has been deprecated starting with Octopus Server 2020.6.  We made this decision because the uptake was low, and Microsoft has stopped supporting the OS Versions we were publishing (Windows [1809](https://docs.microsoft.com/en-us/windows/release-health/status-windows-10-1809-and-windows-server-2019), [1903](https://docs.microsoft.com/en-us/lifecycle/announcements/windows-10-1903-end-of-servicing), and [1909](https://docs.microsoft.com/en-us/windows/release-health/status-windows-10-1909)).  Going forward, we will only publish the Octopus Server Linux Container.  
+
+:::hint
+We will continue to publish Windows Docker images for Tentacle. Once we've updated the images for Tentacle to Windows 2019 and Windows 20H2, we will deprecate the existing Windows 1809/1903/1909 containers.
```

Comment:
Strong chance that we wont support Windows 20H2. It's... not being stable. 
Can we tweak to say "more modern OS versions" instead?

Reply:
Updated to:
> We will continue to publish Windows Docker images for Tentacle. Once we've updated the Windows images for Tentacle to more modern OS versions, we will deprecate the existing Windows 1809/1903/1909 images.